x-pack/plugins/security_solution/public/detections/components/alerts_table/default_config.tsx

```diff
@@ -95,12 +106,12 @@ export const buildThreatMatchFilter = (showOnlyThreatIndicatorAlerts: boolean):
             alias: null,
             disabled: false,
             negate: false,
-            key: 'signal.rule.threat_mapping',
+            key: 'signal.rule.threat_mapping', // TODO: This needs to be updated to kibana.alert?
```

Comment:
@marshallmain

Reply:
The `signal.rule.threat_mapping.*` are in the aad mappings already individually. We can't alias the `threat_mapping` object as a whole because aliases aren't allowed on objects. :disappointed: